virttest/tests/unattended_install.py

```diff
@@ -1216,6 +1217,21 @@ def attempt_to_log_useful_files(test, vm):
                             fd_dst.write("Unknown exception while getting "
                                          "content: %s" % details)
                             failures = True
+            for cmd in ["journalctl --no-pager"]:
+                dst = os.path.join(test.outputdir, vm.name, str(i),
+                                   astring.string_to_safe_path(cmd))
+                with open(dst, 'w') as fd_dst:
+                    try:
+                        fd_dst.write(console.cmd(cmd))
+                        logging.info('Attached "%s" cmd output at "%s"',
+                                     cmd, dst)
+                    except Exception as details:
+                        logging.warning("Unknown exception while "
+                                        "attempt_to_log_useful_files(): "
```

Comment:
I was going to suggest you to get the function's name dynamically, but after a googled at glance I realized it is not trivial for just a cosmetic thing. So never mind...